purchases/src/main/kotlin/com/revenuecat/purchases/google/BillingWrapper.kt

```diff
@@ -78,6 +77,7 @@ internal class BillingWrapper(
     @Suppress("unused")
     private val diagnosticsTrackerIfEnabled: DiagnosticsTracker?,
     private val dateProvider: DateProvider = DefaultDateProvider(),
+    private val queryProductDetailsUseCase: QueryProductDetailsUseCase,
```

Comment:
Should we add a default value so we don't need need to pass it from the constructor everywhere else?